src/AmqpBundle/Sandbox/NullChannel.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace M6Web\Bundle\AmqpBundle\Sandbox;
+
+use AMQPConnection;
+
+/**
+ * Channel that does not anything with connection
```

Comment:
I think the correct sentence should be: 

`Channel that does not do anything with connection`